webpack/assets/javascripts/react_app/redux/API/APIMiddleware.js

```diff
@@ -1,11 +1,17 @@
+/* eslint-disable no-console */
 import { API_OPERATIONS, actionTypeGenerator } from './';
 import { get } from './APIRequest';
+import { startPolling } from './APIActions';
 
 export const APIMiddleware = store => next => action => {
```

Comment:
How about making a new `Interval` middleware, so the APIMiddleware will handle the requests and the interval will trigger an Action interval?

Reply:
I realy like the idea and I will investigate more about it,
What other intervals do we need?
Is it possible that using more middlewars will slow down performance as they are getting called by every action?

Reply:
> What other intervals do we need?

Can't think about any but IMO if we can make it more reusable and independent let's go for it :+1: 

> Is it possible that using more middlewars will slow down performance as they are getting called by every action?

IMO it should not affect performance at all. (if the condition will be simple as it should)